python3Pyside2QML/src/mainwindow.qml

```diff
@@ -1,21 +1,35 @@
-import QtQuick 2.0
-import QtQuick.Controls 2.0
+import QtQuick 2.13
+import QtQuick.Window 2.13
 
-Rectangle {
-    width: 200
-    height: 200
-    color: "green"
+Window {
+    id: mainWindow
+    visible: true
+    visibility: "FullScreen"
```

Comment:
Please change to `visibility: "Windowed"`, because FullScreen occupies the entire screen and you can't use VSCode with the app running if you have just one screen.